### DISCUSSION TITLE
What is the 'suggest response and read' button for ?

### DISCUSSION THREAD

**Discussion by tan-banana** [vivekuppal/transcribe#182] [Q&A] [answered]
suggest response now  = same output as ' suggest response and READ" . But what does read mean? Read the whole transcript ?

**Comment by vivekuppal** [comment, marked as answer]
Suggest response and read is intended to be used for a conversational mode. 

Use cases
1. Interact with AI using voice
2. Learn a new language

Follow these steps

Disable transcription from speakers.
Ask a question and ensure it is correctly captured in the transcription window.
Use the response and read button.
AI will respond to the question. The response is presented as text and the response is also read out aloud
